modules/navier_stokes/include/base/NSFVBase.h

```diff
@@ -1060,6 +1069,8 @@ NSFVBase<BaseType>::NSFVBase(const InputParameters & parameters)
     _energy_two_term_bc_expansion(parameters.get<bool>("energy_two_term_bc_expansion")),
     _passive_scalar_two_term_bc_expansion(
         parameters.get<bool>("passive_scalar_two_term_bc_expansion")),
+    _use_standard(parameters.get<bool>("use_standard")),
+    _use_superficial(parameters.get<bool>("use_superficial")),
```

Comment:
Even if they do set them it won't produce any errors?

Reply:
Not setting these parameters in the action will result in error, because they will be required by the friction kernel. Setting them without the friction error will have no effect on the simulation.

Reply:
Not setting these parameters won't produce an error because there is a default associated to them. Even if you got an error it would not be useful for the user considering that it would point to objects which are under the hood and invisible to them. So I'd recommend adding the error checks at the action level.

Reply:
I added checks in the action for the case where we define friction. If we have friction the user needs to specify the friction formulation, and if we have porous formulation the user needs to define if they are gonna use superficial values or interstitial values.